metadata-ingestion-modules/airflow-plugin/src/datahub_airflow_plugin/_config.py

```diff
@@ -68,11 +71,19 @@ class DatahubLineageConfig(ConfigModel):
 
     disable_openlineage_plugin: bool = True
 
-    def make_emitter_hook(self) -> "DatahubGenericHook":
+    def make_emitter_hook(self) -> Union["DatahubGenericHook", "DatahubCompositeHook"]:
```

Comment:
just a thought, not necessary to do: should the `DatahubGenericHook` be the thing responsible for constructing the composite emitter?

e.g. you can pass it either a string or a list of strings, and it does the right thing